test/inductor/test_aot_inductor.py

```diff
@@ -1202,6 +1240,7 @@ class AOTInductorTestABICompatibleCuda(TestCase):
     # test_failures, xfail by default, set is_skip=True to skip
     {
         "test_dup_unbacked_sym_decl": TestFailure(("abi_compatible_cuda",)),
+        "test_missing_cubin": TestFailure(("abi_compatible_cuda",)),
```

Comment:
Can we add a comment on what happened? The `# There is a double-free issue which will be fixed in another PR` at the bottom is pretty informative, and people know it will be fixed. It's hard to tell for other tests whether the XFAIL is in purpose or it's need to be fixed?